CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# Upcoming Release 
+# Upcoming Release
+
+## Bug Fixes:
+
+- Fixes bug where tabs selected attribute not working if manually change tab. Issuse #3040
```

Comment:
Thanks @tomchang25 !

You need to add the github link, e.g. [PR 3055](https://github.com/gradio-app/gradio/pull/3055).

Do you mind moving this to the `## Bug Fixes` section below?

Reply:
Also, can you revert all the other changes to this file except adding the line for this PR?